docs/experiments-ance.md

```diff
@@ -0,0 +1,105 @@
+# Pyserini: Replicating Microsoft's ANCE Results
+
+This guide provides replication instructions for the following dense retrieval work:
+
+> Lee Xiong, Chenyan Xiong, Ye Li, Kwok-Fung Tang, Jialin Liu, Paul Bennett, Junaid Ahmed, Arnold Overwijk. [Approximate Nearest Neighbor Negative Contrastive Learning for Dense Text Retrieval](https://arxiv.org/pdf/2007.00808.pdf)
+
+You'll need a Pyserini [development installation](https://github.com/castorini/pyserini#development-installation) to get started.
+
+
+## MS MARCO Passage
+Download the query encoder checkpoint:
+```bash
+$ wget https://www.dropbox.com/s/u02glpszk3jv6ws/ance-msmarco-passage-encoder.tar.gz
+$ tar -xvf ance-msmarco-passage-encoder.tar.gz
+```
+
+**ANCE retrieval** with brute-force index:
+```bash
+$ python -m pyserini.dsearch --topics msmarco-passage-dev-subset \
+                             --index msmarco-passage-ance-bf \
+                             --encoder ance-msmarco-passage-encoder \
+                             --batch-size 36 \
+                             --threads 12 \
+                             --output runs/run.msmarco-passage.ance.bf.tsv \
+                             --msmarco
+```
+To evaluate:
+```bash
+$ python tools/scripts/msmarco/msmarco_passage_eval.py tools/topics-and-qrels/qrels.msmarco-passage.dev-subset.txt runs/run.msmarco-passage.ance.bf.tsv
```

Comment:
Actually, let's change to `python -m pyserini.eval.msmarco_passage_eval ...`

Reply:
... and below, also.

Reply:
seems all other experiments doc except `pypy-replication.md` are using evaluation scripts from `tools` rn.
Besides, dpr evaluation scripts haven't moved to `pyserini` yet. 
Let's do the things all together in a separate PR?

Reply:
yup, ahead of you: https://github.com/castorini/pyserini/pull/394